source/reference/minio-mc/mc-ilm-tier-add.rst

```diff
@@ -0,0 +1,343 @@
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc ilm tier add TIER_TYPE                    \
+                         TARGET                       \
+                         TIER_NAME                    \
+                         --endpoint string            \
+                         [--region string]            \
+                         [--access-key value]         \
+                         [--secret-key value]         \
+                         [--use-aws-role]             \
+                         [--account-name value]       \
+                         [--account-key value]        \
+                         [--credentials-file value]   \
+                         --bucket value               \
+                         [--prefix value]             \
+                         [--storage-class value]
+
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+
+Parameters
+~~~~~~~~~~
+
+The command accepts the following arguments:
+
+.. mc-cmd:: TIER_TYPE
+   :required:
+
+   The Cloud Service Provider storage backend ("Tier") to which MinIO transitions objects. 
+   Specify *one* of the following supported values:
+
+   .. list-table::
+      :stub-columns: 1
+      :width: 100%
+      :widths: 30 70
+
+      * - ``minio``
+        - Use a remote MinIO deployment as the storage backend for the new Tier.
+
+          Requires specifying the following additional options:
+
+          - :mc-cmd:`~mc ilm tier add --access-key`
+          - :mc-cmd:`~mc ilm tier add --secret-key`
+
+      * - ``s3``
+        - Use AWS S3 as the storage backend for the new Tier.
+
+          Requires specifying the following additional options:
+
+          - :mc-cmd:`~mc ilm tier add --access-key`
+          - :mc-cmd:`~mc ilm tier add --secret-key`
+
+      * - ``azure``
+        - Use :abbr:`Azure (Microsoft Azure)` Blob Storage as the storage backend for the new Tier.
+
+          Requires specifying the following additional options:
+
+          - :mc-cmd:`~mc ilm tier add --account-name`
+          - :mc-cmd:`~mc ilm tier add --account-key`
+         
+      * - ``gcs`` 
+        - Use :abbr:`GCP (Google Cloud Platform)` Cloud Storage as the storage backend for the new Tier.
+
+          Requires specifying the following additional option:
+
+          - :mc-cmd:`~mc ilm tier add --credentials-file`
+
+.. mc-cmd:: TARGET
+   :required:
+
+   The :mc-cmd:`alias <mc alias>` of a configured MinIO deployment on which the command creates the new remote tier.
+   Rules on this deployment use the new tier to transition objects from this deployment to a remote location.
+      
+.. mc-cmd:: TIER_NAME
+   :required:
+
+   The name to associate with the new remote tier. 
+   The name **must** be unique across all configured tiers on the MinIO cluster.
+
+   You **must** specify the tier in all-caps, e.g. ``WARM_TIER``.
```

Comment:
@krisis I've wondered if there's a way for us to autocap this so we always do the right thing. Not sure if that's more disruptive than asking users to hold down SHIFT or use caps-lock.